app/Http/Controllers/SettingsController.php

```diff
@@ -869,7 +869,7 @@ public function postLdapSettings(Request $request)
         $setting->ldap_server             = $request->input('ldap_server');
         $setting->ldap_server_cert_ignore = $request->input('ldap_server_cert_ignore', false);
         $setting->ldap_uname              = $request->input('ldap_uname');
-        if (Input::has('ldap_pword')) {
+        if ($request->input('ldap_pword') !== '') {
```

Comment:
This is another nice cleanup that I still think should go in a different commit.

Reply:
Reverted change. Will open a new PR.